ui/app/AppLayouts/Profile/views/MessagingView.qml

```diff
@@ -373,20 +380,7 @@ SettingsContentBase {
                             StatusSwitch {
                                 id: siteSwitch
                                 checked: !!model.isWhitelisted
-                                onCheckedChanged: {
-                                    let settings = localAccountSensitiveSettings.whitelistedUnfurlingSites
-
-                                    if (!settings) {
-                                        settings = {}
-                                    }
-
-                                    if (settings[address] === this.checked) {
-                                        return
-                                    }
-
-                                    settings[address] = this.checked
-                                    localAccountSensitiveSettings.whitelistedUnfurlingSites = settings
-                                }
+                                onCheckedChanged: RootStore.updateWhitelistedUnfurlingSites(model.address, checked)
```

Comment:
Actually here we want to reflect changes from `model.isWhitelisted` so that when a tenor warning is accepted we get the already loaded Messageingview in settings updated.